### DISCUSSION TITLE
Bots ignore hostile relations at outpost when ordered to deconstuct

### DISCUSSION THREAD

**Discussion by Instant2116** [FakeFishGames/Barotrauma#13835] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When you're at outpost and outpost is hostile every crewmember will be attacked. But Outpost still has a deconstructor and bots are willingly go to outpost with item to deconstruct and die. Player also can't order "don't touch this" to outpost's deconstructor as this will require a suicide run (into hostile outpost and back.)

### Reproduction steps

1. Enter hostile outpost will working deconstructor 
2. Make sure there is no an available deconstructor at your submarine
3. Order bot to deconstruct an item
4. Observe an "absolute duty" as bot will sacrifice himself to deconstruct a item 

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
[Finally addressed in our private repo](https://github.com/FakeFishGames/Barotrauma-development/commit/6b0616b0d9b9ffbbd1120a1db5ff1f4feb3aae80), fix coming up in the next unstable build. Sorry for it taking so ridiculously long for us to look into this!
